mozdownload/scraper.py

```diff
@@ -863,7 +862,6 @@ def platform_regex(self):
         PLATFORM_FRAGMENTS = {'linux': 'linux',
                               'linux64': 'linux64',
                               'mac': 'macosx',
-                              'mac64': 'macosx64',
```

Comment:
@whimboo 

Maybe I misread your statement from the issue (link #215). I read it as: "Remove mac64 option"... so I did ;)

I did not give the option in the regex of to download either macosx or macosx64 because I thought this might lead to a bug or inaccuracies (not possible to download a specific build because of it...)